templates/packer/ansible/roles/common/files/worker-initialization-scripts/worker_executor.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+# Subscribe the executor, this can be removed once the worker functions as a proxy for all sources.
+SUBSCRIPTION_MANAGER_COMMAND_ARN=$(
+  secretsmanager list-secrets \
+  --filters Key=name,Values=executor-subscription-manager-command \
+  --query SecretList[0].ARN --output text
+)
+
+/usr/local/bin/aws secretsmanager get-secret-value \
+  --secret-id "${SUBSCRIPTION_MANAGER_COMMAND_ARN}" | jq -r ".SecretString" > /tmp/subscription_manager_command.json
+jq -r ".subscription_manager_command" /tmp/subscription_manager_command.json | bash
+rm -f /tmp/subscription_manager_command.json
+
+echo "Starting osbuild-jobsite-builder."
+/usr/libexec/osbuild-composer/osbuild-jobsite-builder &
```

Comment:
I wanted a separate service for the builder, so we don't have to run it in background and disown it. Now we can just run it as a part of the system service. Since it's a *oneshot* one, it will be run just once, which is AFAIK exactly what we want. :) IOW, I suggest removing `&` and disown. :)

Reply:
Oh ok, yea that makes total sense!